HousingSearchApi/V1/Infrastructure/Core/FilterQueryBuilder.cs

```diff
@@ -47,6 +47,15 @@ public IQueryBuilder<T> WithWildstarQuery(string searchText, List<string> fields
 
             return this;
         }
+        public IFilterQueryBuilder<T> WithWildstarBoolQuery(string searchText, List<string> fields, int? minimumShouldMatch = 1, TextQueryType textQueryType = TextQueryType.MostFields)
```

Comment:
Is `minimumShouldMatch` used in the implementation at all?

Reply:
Removed - was part of an earlier implementation.